src/schema.graphql

```diff
@@ -284,6 +287,12 @@ type Vault @entity {
   "The vault version"
   version: BigInt!
 
+  "The vault osToken LTV percent"
+  ltvPercent: BigInt
+
+  "The vault osToken liquidation threshold percent"
```

Comment:
Please add comment that if the value is null, the value from the OsTokenConfig should be used.